frontend/src/app/page.tsx

```diff
@@ -86,13 +87,26 @@ const BountyList: FC = () => {
 };
 
 const Home: FC = () => {
+    const [hasConnectedAccount, setHasConnectedAccount] = useState(false);
+    const { address, connector, isConnected } = useAccount()
+    
+    useEffect(() => {
+        if ((isConnected)&&(address)&&(connector)) {
+            setHasConnectedAccount(true);
```

Comment:
Instead of `if (condition) set(true) else set(false)`, you can do `set(condition)`.

Reply:
Done!